### PR TITLE
Correctly invalidating browser.js and i18n friends

### PR DESCRIPTION
**Pull request by diunko**
Uses features available in bem-tools with this https://github.com/bem/bem-tools/pull/408 patch.

### CODE REVIEW

File: .bem/techs/bemhtml.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-    compat = require('bemhtml-compat');
+    compat = require('bemhtml-compat'),
 
-exports.techMixin = {
+    Tech = require('bem/lib/tech').TechV2;
 
-    getSuffixes : function() {
-        return ['bemhtml', 'bemhtml.xjst'];
-    },
+exports.baseTech = Tech;
```
Comment (tadatuta) [inline .bem/techs/bemhtml.js]: Арикон сказал, что вместо `baseTech` нужно делать `exports.API_VER = 2;`

File: .bem/techs/browser.js+bemhtml.js [hunk 2]
```diff
@@ -1,70 +1,64 @@
...
-
-                                // and return new result
-                                return res;
-
-                            });
-
+    API_VER:2,
+
+    getWeakBuildSuffixesMap:function(){
+        return {"js":["browser.js","vanilla.js","js","bemhtml"]};
```
Comment (tadatuta) [inline .bem/techs/browser.js+bemhtml.js]: мы обычно форматируем код как-то так:

``` javascript
return { js: ['browser.js', 'vanilla.js', 'js', 'bemhtml'] };
```

File: .bem/techs/browser.js+bemhtml.js [hunk 3]
```diff
@@ -1,70 +1,64 @@
...
             });
-
+    },
+
+    getBuildResult:function(files, suffix, output, opts){
+
+        var bemhtmlTech = this.context.createTech("bemhtml");
+        var browserTech = this.context.createTech("browser.js");
+
+        var decl = this.transformBuildDecl(this.context.opts.declaration);
```
Comment (tadatuta) [inline .bem/techs/browser.js+bemhtml.js]:
``` javascript
var bemhtmlTech = this.context.createTech('bemhtml'),
    browserTech = this.context.createTech('browser.js'),
    decl = this.transformBuildDecl(this.context.opts.declaration);
```

File: .bem/techs/browser.js+bemhtml.js (hunk 2 above)
Comment (diunko) [inline .bem/techs/browser.js+bemhtml.js]: Массивы прилегают, объекты отстоят. Ок.

**Comment by arikon** [comment]
@diunko Для чего код в `common.bundles/index/blocks`?

**Comment by diunko** [comment]
@arikon that's a poor man's testing mode being on. Though, it shouldn't be there any longer.

**Comment by diunko** [comment]
@arikon no, wait. Looks like it's ok, because that's just what other folks do. Guess it'll go to some other nice place when technology tests arrive.

**Comment by arikon** [comment]
Close in favor of #239

**Pull request closed without merging**